examples/quick-start-guide.md

```diff
@@ -0,0 +1,109 @@
+# Quick Start Guide
+
+This section helps you to quickly set up and run Ballerina so that you can use it for various solutions. Here we 
+
+## Install Ballerina
+
+1. [Download](https://ballerina.io/downloads) Ballerina for your Operating System. 
+1. Follow the instructions given in the [Ballerina Getting Started page](#https://ballerina.io/learn/getting-started/) to set it up. 
+
+> **Note**: Throughout this documentation, `<ballerina_home>` refers to the directory in which you just installed Ballerina.
+
+## Start a Project, Run a Service, and Invoke It
+
+Start your project by navigating to a directory of your choice and running the following command.
+
+```bash
+$ ballerina init
+```
+
+You see a response confirming that your project is initialized. This automatically creates a typical Hello World service for you in your directory. A Ballerina service represents a collection of network accessible entry points in Ballerina. A resource within a service represents one such entry point. The generated sample service exposes a network entry point on port 9090.
+
+You can run the service using the `ballerina run` command.
+
+```bash
+$ ballerina run hello_service.bal
+```
+
+You get the following output.
+
+```bash
+ballerina: initiating service(s) in 'hello_service.bal'
+ballerina: started HTTP/WS endpoint 0.0.0.0:9090
+```
+
+This means your service is up and running. You can invoke the service using an HTTP client. In this case, we use cURL.
+
+```bash
```

Comment:
IMO it would be best if we can ask them to copy and paste the URL in their favorite browser and see the response. WDYT? In MS Windows, they don't have curl pre-installed. Alternatively, mention the curl command and say that we will be using curl in our upcoming example and you can use curl or any other tool like Postman to execute the requests.